src/execution/execute-afm.convert.ts

```diff
@@ -0,0 +1,75 @@
+// (C) 2007-2019 GoodData Corporation
+
+import { AFM, ExecuteAFM } from '@gooddata/typings';
+import { unset, cloneDeep } from 'lodash';
```

Comment:
As far as I'm aware of this will cause that whole lodash lib will be bundled to final package... to avoid that its better to import only a required lodash function...

`import unset = require('lodash/unset');` etc...

Reply:
good to know, wasn't aware of that; fixing